tuist/tuist
# Fix hash for synthetized Bundle+Target.swift

### Short description 📝

This PR resolves https://github.com/tuist/tuist/issues/3984
The issue was that we were not generating `contentHash` up front for the synthesized  `Bundle+Target.swift` as we do in `SynthesizedResourceInterfaceProjectMapper` for other files. It is important to notice we don't need to generate those files when loading the graph (nor for getting the hashes)

### How to test the changes locally 🧐

Unit tests are updated, an example of how to try it out locally is to run `tuist tests` inside `ios_app_with_framework_and_resources` (make sure to not have run `tuist generate` before)

### Checklist ✅
